test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -1340,6 +1360,37 @@ namespace iroha {
           });
     }
 
+    /**
+     * This test checks that tables data is sorted as integrals and not as text
+     * @given initialized storage with 10 blocks, permissioned account
+     * @when get account transactions with first_tx_hash offset to get the last
+     * tx when page_size is more than one
+     * @then Return only one (the last) transaction
+     */
+    TEST_F(GetAccountTransactionsExecutorTest, ValidPaginationOrder) {
+      addPerms({shared_model::interface::permissions::Role::kGetMyAccTxs});
+
+      commitBlocks();
+      commitAdditionalBlocks(kTxPageSize);
+
+      auto query =
+          TestQueryBuilder()
+              .creatorAccountId(account_id)
+              .getAccountTransactions(account_id, kTxPageSize, hashes.back())
+              .build();
+      auto result = executeQuery(query);
+      checkSuccessfulResult<shared_model::interface::TransactionsPageResponse>(
+          std::move(result), [](const auto &cast_resp) {
+            ASSERT_EQ(cast_resp.transactions().size(), 1);
```

Comment:
why assert and not expect? your code seems apt to continue even when this cheek fails.